2-0-data-structures-and-algorithms/2-2-6-binary-search-tree/src/test/java/com/bobocode/cs/BinarySearchTreeTest.java

```diff
@@ -260,7 +260,21 @@ void inorderTraversal() {
                  * .........\
                  * ..........5
                  */
-                arguments(new Integer[]{1, 2, 3, 4, 5}, 4));
+                arguments(new Integer[]{1, 2, 3, 4, 5}, 4),
+                /*
+                 * ......10
+                 * ...../..\
+                 * ....6....18
+                 * .../......\
+                 * ..4........20
```

Comment:
@RepNazar I just realized that **8, and 12 are missing**

Reply:
Also noticed, that some "Always go right" implementation will pass all tests including mine. So I changed it